data/builtInFeatures.js

```diff
@@ -0,0 +1,114 @@
+// https://github.com/zloirock/core-js
+
+module.exports = {
+  // es2015
+  // core-js/fn/map
+
+  // "es6.typed/array-buffer": "typed arrays / ",
+  "es6.typed.data-view": "typed arrays / DataView",
+  "es6.typed.int8-array": "typed arrays / Int8Array",
+  "es6.typed.uint8-array": "typed arrays / Uint8Array",
+  "es6.typed.uint8-clamped-array": "typed arrays / Uint8ClampedArray",
+  "es6.typed.int16-array": "typed arrays / Int16Array",
+  "es6.typed.uint16-array": "typed arrays / Uint16Array",
+  "es6.typed.int32-array": "typed arrays / Int32Array",
+  "es6.typed.uint32-array": "typed arrays / Uint32Array",
+  "es6.typed.float32-array": "typed arrays / Float32Array",
+  "es6.typed.float64-array": "typed arrays / Float64Array",
+
+  "es6.map": "Map",
+  "es6.set": "Set",
+  "es6.weak-map": "WeakMap",
+  "es6.weak-set": "WeakSet",
+  // proxy not implemented
+
+  "es6.reflect.apply": "Reflect / Reflect.apply",
+  "es6.reflect.construct": "Reflect / Reflect.construct",
+  "es6.reflect.define-property": "Reflect / Reflect.defineProperty",
+  "es6.reflect.delete-property": "Reflect / Reflect.deleteProperty",
+  "es6.reflect.get": "Reflect / Reflect.get",
+  "es6.reflect.get-own-property-descriptor": "Reflect / Reflect.getOwnPropertyDescriptor",
+  "es6.reflect.get-prototype-of": "Reflect / Reflect.getPrototypeOf",
+  "es6.reflect.has": "Reflect / Reflect.has",
+  "es6.reflect.is-extensible": "Reflect / Reflect.isExtensible",
+  "es6.reflect.own-keys": "Reflect / Reflect.ownKeys",
+  "es6.reflect.prevent-extensions": "Reflect / Reflect.preventExtensions",
+  "es6.reflect.set": "Reflect / Reflect.set",
+  "es6.reflect.set-prototype-of": "Reflect / Reflect.setPrototypeOf",
+
+  "es6.promise": "Promise",
+  "es6.symbol": "Symbol",
+
+  "es6.symbol": "well-known symbols / Symbol.hasInstance",
```

Comment:
`@@hasInstance` - works with `babel` optional transform + `core-js`
`@@isConcatSpreadable` - can be polyfilled, but for some performance reason I didn't add it before. Maybe if I'll find some time in the future...
`@@iterator` - works with `core-js` and some `babel` transforms.
`@@match` - works with `core-js`
`@@replace` - works with `core-js`
`@@search` - works with `core-js`
`@@species` - in some places works with `core-js`
`@@split` - works with `core-js`
`@@toPrimitive` - can't be completely polyfilled / transpiled
`@@toStringTag` - works with `core-js`
`@@unscopables` - theoretically, can be transpiled, but it should work in sloppy mode only

from zloirock on slack